routes/stripe.js

```diff
@@ -96,7 +96,7 @@ var stripeRoutes = {
     try {
       event = stripe.webhooks.constructEvent(payload, signature, endpointSecret);
     } catch (constructEventErr) {
-      event = [null, constructEventErr];
+      console.error('Error: ', constructEventErr);
```

Comment:
Can we make it `"constructEvent Error:"`, that way it's more easily surfaced in the logs when searching